src/lib/apis/maxim.ts

```
@@ -1,109 +1,224 @@
+			validateStatus: (status) => status < 600, // Don't throw on any status code, let us handle it
+		});
+
+		// Configure comprehensive retry logic
+		axiosRetry(this.axiosInstance, {
+			retries: 5, // Maximum retry attempts
+
+			// Exponential backoff with jitter to prevent thundering herd
+			retryDelay: (retryCount, error) => {
+				// Respect Retry-After header if present
+				const retryAfter = error.response?.headers["retry-after"];
+				if (retryAfter && !isNaN(parseInt(retryAfter))) {
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)